### ISSUE TITLE
Photo is getting worse with the time

### ISSUE THREAD

**Issue by stefan-niedermann** [bitfireAT/davdroid#545]
I have a curious problem. I am using latest DavDroid from F-Droid Repo and latest OwnCloud (but it happend with older OwnCloud and DavDroid-Versions, too).

After some time i notice on some images the quality is getting worse and worse. And every time it gets an tinge of green.

I'm afraid i can not surely say it's caused by DavDroid or OwnCloud and it does not happen with *all* of the pictures, but always with the *same* pictures (i sometimes replace the broken images with "fresh" ones because it looks ugly...)

Do you have any idea if it's possibly caused by DavDroid? I will open and link an Issue in OwnCloud Calendar, too.

This is a broken example picture:
![photo](https://cloud.githubusercontent.com/assets/4741199/8125997/077574e0-10eb-11e5-9a9b-d788d9efe5d8.jpg)

Notice: It's not broken within a second, but every day the picture looses quality. After some days or weeks, it looks like this.

Hopefull, you can help me.

**Comment by rfc2822** [comment]
Is this the same issue as #296?

**Comment by stefan-niedermann** [comment]
yeah, possible. but this issue seems to be closed, so why do i still have the problem?

**Comment by untitaker** [comment]
@stefan-niedermann It's still not clear whether ownCloud has fully fixed this issue, or if it is caused by Android or DavDroid.

Could you check whether swapping any component with an alternative (Radicale instead of ownCloud, CardDAV-Sync instead of DavDroid) fixes this issue? Which ROM are you using?

**Comment by stefan-niedermann** [comment]
I am using Paranoid Android 4.4. Well, swapping all components will cost very much time. I will try to find some minutes/hours, but i can not promise that this happens soon...

**Comment by untitaker** [comment]
It seems Flock (which is IIRC a fork of DavDroid) has a commit related to this: https://github.com/WhisperSystems/Flock/commit/958a76e4f629cf8393722198822fc83033f0cc08

**Comment by rfc2822** [comment]
@untitaker The `skip_processing` field which is used in this commit looks interesting.

**Comment by devvv4ever** [comment]
is this maybe related to https://github.com/owncloud/contacts/issues/994 ?
